tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -727,7 +727,7 @@ ROOT::Experimental::Internal::RMiniFileReader::GetNTupleProper(std::string_view
 
    offset = key.GetSeekKey() + key.fKeyLen;
 
-   constexpr size_t kMinNTupleSize = 70; // size of a RTFNTuple version 4 (min supported version)
+   constexpr size_t kMinNTupleSize = 78; // size of a RTFNTuple version 2 (min supported version)
```

Comment:
I believe the idea is that the anchor size can grow in the future, so we have to maintain `kMinNTupleSize` independently of the `RTFNuple` structure

Reply:
But that contradicts the comment next to it then, no?

Reply:
Hm not sure, for me it's quite clear:
 * For future versions 3, 4, 5, ... `RTFNTuple` gains more members that have a default initialization. That will increase `RTFNuple::GetSizePlusChecksum`
 * This exact line will forever stay like this because it was the size of `RTFNTuple` version 2, when we declared RNTuple 1.0.0.0, and we will always be able to read it back.

Reply:
I'll left the hard-coded 78 but added a static_assert, which makes it clear that this number is currently identical to `RTFNuple::GetSizePlusChecksum()`. If we grow the anchor in the future, the static assert will remind us of this spot.